### PR TITLE
Vertical col support

### PR DESCRIPTION
**Pull request by Nitish1814**

### CODE REVIEW

File: common/client/src/main/java/zingg/common/client/util/verticalDisplay/VerticalDisplayUtility.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    private List<Pair<String, String>> getComparisonPairs(ZFrame<D, R, C> zFrame, String[] columns) {
+
+        ZFrame<D, R, C> zFrame1 = zFrame.limit(1);
+        ZFrame<D, R, C> zFrame2 = zFrame.except(zFrame1);
+
+        R row1 = zFrame.head();
+        R row2 = zFrame2.head();
+
+        List<Pair<String, String>> comparison_pairs = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (sonalgoyal) [reply common/client/src/main/java/zingg/common/client/util/verticalDisplay/VerticalDisplayUtility.java:53]: you dont need an intermediary structure here. collectAsList would give you the List<Row> rows and you can iterate over the columns to build VerticalDisplayModel(df.getAsString(rows[0], column, df.getAsString(rows[1], column)

File: common/core/src/test/java/zingg/common/core/util/TestVerticalDisplayUtility.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+public abstract class TestVerticalDisplayUtility<S, D, R, C> {
+
+    private final DFObjectUtil<S, D, R, C> dfObjectUtil;
+
+    public TestVerticalDisplayUtility(DFObjectUtil<S, D, R, C> dfObjectUtil) {
+        this.dfObjectUtil = dfObjectUtil;
+    }
+
+    @Test
+    public void testWithoutNulls() throws Exception, ZinggClientException {
```
Comment (sonalgoyal) [inline common/core/src/test/java/zingg/common/core/util/TestVerticalDisplayUtility.java:22]: can you also try with an empty dataframe?

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: common/core/src/test/java/zingg/common/core/util/TestVerticalDisplayUtility.java (hunk 2 above)
Comment (Nitish1814) [reply common/core/src/test/java/zingg/common/core/util/TestVerticalDisplayUtility.java:22]: done

File: common/client/src/main/java/zingg/common/client/util/verticalDisplay/VerticalDisplayModel.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package zingg.common.client.util.verticalDisplay;
```
[the remark this answers is not in the document]
Comment (Nitish1814) [reply common/client/src/main/java/zingg/common/client/util/verticalDisplay/VerticalDisplayModel.java]: done

**Pull request merged**